Classes/Service/Reporting.php

```diff
@@ -104,7 +104,13 @@ public function getNodeStat(NodeInterface $node, ControllerContext $controllerCo
 
         $nodeUri = $this->getLiveNodeUri($node, $controllerContext);
         $hostname = !empty($siteConfiguration['overrideHostname']) ? $siteConfiguration['overrideHostname'] : $nodeUri->getHost();
-        $filters = 'ga:pagePath==' . $nodeUri->getPath() . ';ga:hostname==' . $hostname;
+        $pagePathPrefix = !empty($siteConfiguration['pagePathPrefix']) ? $siteConfiguration['pagePathPrefix'] : '';
+        $includePagePath = !isset($siteConfiguration['includePagePath']) || $siteConfiguration['includePagePath'];
+
+        $filters = 'ga:hostname==' . $hostname;
+        if ($includePagePath) {
+            $filters .= ';ga:pagePath==' . $pagePathPrefix . $nodeUri->getPath();
```

Comment:
If I understand correctly the node uri is incomplete. That means the problem actually happens in `$this->getLiveNodeUri`. 
So we should make sure it generates the correct Uris. And then I wonder, why the generated Uris are not correct. Then you would have incomplete Uris everywhere?

Reply:
The node URI created by Neos is just right and fits the actual uri of the page. 
The problem is that the uri in Google Analytics has an additional prefix.